src/io/fabric8/Fabric8Commands.groovy

```diff
@@ -988,8 +988,8 @@ def openShiftImageStreamInstall(String name, String location) {
 @NonCPS
 def setupGitSSH() {
     sh """
-       git config user.email fabric8-admin@googlegroups.com
-       git config user.name fabric8-release
+       git config --global user.email fabric8-admin@googlegroups.com
```

Comment:
@rupalibehera could you please elaborate (as it is missing in the commit message) why we want to set the config globally?

Reply:
@sthaha I've seen both global and local in the previous code, this is just making the sore thumb standout now. This code is slightly safer to execute, like before a git clone.

@rupalibehera Ideally add a 'why' as well to the commit message.

Reply:
below is the error mesaage which I was seeing, hence the above change 
```

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'root@jenkins-slave-pf6x7-83gmp.(none)')
```
https://jenkins.cd.test.fabric8.io/job/fabric8-ui/job/fabric8-ui/job/master/513/console